src/Catty/DataModel/UserVariable/UserVariable.swift

```diff
@@ -20,8 +20,15 @@
  *  along with this program.  If not, see http://www.gnu.org/licenses/.
  */
 
+protocol UserVariableProtocol: CBMutableCopying {
```

Comment:
Please rename to _UserDataProtocol_ as suggested in the ticket and create a new file. This protocol will be part of a major refactoring.